res/skins/Tango/deck_overview.xml

```diff
@@ -43,13 +43,13 @@ Variables:
     </MarkRange>
     <Mark>
       <Control>loop_start_position</Control>
-      <Text>[&nbsp;</Text>
+      <Text>[&#x0A;</Text>
```

Comment:
Please add a comment here, that we want &nbsp and why we can't use it and probably why we need it here.

I think we need to choose 
&#xa0 or &#160

Reply:
Whoops, sorry, I must have switched the numbers by mistake. I fixed up the commit.